package/MDAnalysis/core/universe.py

```diff
@@ -321,6 +325,13 @@ def __init__(self, *args, **kwargs):
 
             if not coordinatefile:
                 coordinatefile = None
+            # parse transformations
```

Comment:
I'm not sure we need all this crud here.  Will it work if you just do:

```python
self.load_new()  # this populates self.traj
trans_arg = kwargs.whatever()
if trans_arg:
    transforms = [trans_arg] if callable(trans_arg) else trans_arg
    self.trajectory.add_transformations(transforms)
```

then all the transformation stuff is done in the same place